packages/shared/src/components/cards/ActionsButtons/ActionButtons.tsx

```diff
@@ -119,4 +125,137 @@ export default function ActionButtons({
       {lastActions}
     </div>
   );
-}
+};
+
+const ActionButtonsDownvote: typeof ActionButtonsControl = ({
```

Comment:
There were too many changes to margins/paddings to it was cleaner to just create separate component and utilize `withExperiment` HOC